stellar/txobserver/main.go

```diff
@@ -0,0 +1,146 @@
+package txobserver
+
+import (
+	"net/http"
+	"time"
+
+	"github.com/stellar/go/clients/horizonclient"
+	slog "github.com/stellar/go/support/log"
+	"github.com/stellar/starbridge/store"
+)
+
+type Observer struct {
+	Client *horizonclient.Client
+	Store  *store.Memory
+}
+
+func (o *Observer) Run() error {
+	log := slog.DefaultLogger.WithField("service", "stellar_txobserver")
+
+	root, err := o.Client.Root()
+	if err != nil {
+		log.Fatal("Unable to access Horizon root resource")
+	}
+
+	ledgerSequence := uint32(root.HorizonSequence)
```

Comment:
I think we want to be fetching / recording the last ledger we ingested in the db